### PR TITLE
fix wheel build

### PR DESCRIPTION
**Pull request by horheynm**
Import is_release from version.py for setup.py.
Running gdb, _generate_version didnt recognize global variables, so made sure the necessary variables are in scope.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,18 +16,25 @@
...
+    version_major,
+    version_minor,
+    version_bug,
+) = get_release_and_version(package_path)
 
 # load and overwrite version and release info from sparseml package
 exec(open(os.path.join("src", "sparseml", "version.py")).read())
 print(f"loaded version {version} from src/sparseml/version.py")
-version_nm_deps = f"{version_major_minor}.0"
+version_nm_deps = f"{version}.0"
```
Comment (dhuangnm) [inline setup.py:37]: This line should not change since version is the full version string?

Comment (horheynm) [reply setup.py:37]: Is this the different than line 25 on the old code?
`version_major_minor = version`?

Comment (dhuangnm) [reply setup.py:37]: From the old code you can see that it's using major.minor.0, but here in the new code {version} is the full string, so it becomes major.minor.hotfix.0?

File: src/sparseml/version.py [hunk 2]
```diff
@@ -45,7 +49,7 @@ def _generate_version():
     "version_build",
     "version_major_minor",
 ]
-__version__ = _generate_version()
+__version__ = _generate_version(is_release, is_dev, version_base, dev_number)
```
Comment (dhuangnm) [inline src/sparseml/version.py:52]: Do we need to pass these variables? I think they have been defined in the first part of this file?

Comment (horheynm) [reply src/sparseml/version.py:52]: Yeah we need those parameters, if not they break

**Review by bfineran — APPROVED** [review]

**Pull request merged**